scripts/deploy-helm.sh

```diff
@@ -209,6 +209,10 @@ if [ -z "$KUBERNETES_NAMESPACE" ]; then
   KUBERNETES_NAMESPACE="dynatrace"
 fi
 
+if [ -z "$USE_EXISTING_ACTIVE_GATE" ]; then
+  USE_EXISTING_ACTIVE_GATE=true
```

Comment:
There's a similar situation with 
`if [[ $DEPLOYMENT_TYPE != "metrics" ]] && [[ $USE_EXISTING_ACTIVE_GATE != "true" ]]; then`

We can build a log viewer URL in case of using the existing  AG. (case when the existing AG is a Cluster AG, not an environment AG).

Reply:
> @joaquinfilipic-dynatrace Sorry for looking at it so late ;) . The **USE_EXISTING_ACTIVE_GATE** is a read-only variable. Also IMHO for GCP there's nothing to do in the terms of APM-377836, Since as you said the value is used only for k8s deployment and this deployment uses helm chart **values.yaml** that has already `.Values.activeGate.useExisting` set to `true`. So this change should be reverted. @Dawid-Kaszubski

You're right, thanks, I missed it. I suppose that as the new AG is created in case that variable is false explicitly, it didn't make a difference anyway that this change is not setting it.

Reply:
> Another thing that I think is that we have a bug in the deploy-helm.sh. `if [[ $DEPLOYMENT_TYPE == all ]] || [[ $DEPLOYMENT_TYPE == metrics ]] || [[ ($DEPLOYMENT_TYPE == logs && $USE_EXISTING_ACTIVE_GATE == false) ]]; then` the last condition isn't a valid one anymore since it's possible to deploy only logs and use the cluster AG. ( `USE_EXISTING_ACTIVE_GATE = true` )

This I wouldn't know. Did it change at some point, since you said it's not valid "anymore"? In that case, basically I should remove the conditional and make the subsequent validations every time?

Reply:
> There's a similar situation with `if [[ $DEPLOYMENT_TYPE != "metrics" ]] && [[ $USE_EXISTING_ACTIVE_GATE != "true" ]]; then`
> 
> We can build a log viewer URL in case of using the existing AG. (case when the existing AG is a Cluster AG, not an environment AG).

And for this one, leave only 'if [[ $DEPLOYMENT_TYPE != "metrics" ]]'?